components/content/inspira/ui/hyper-text/HyperText.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="overflow-hidden py-2 flex cursor-default scale-100" @mouseenter="triggerAnimation">
+        <div class="flex">
+            <span
+                v-for="(letter, i) in displayText"
+                :key="i"
+                :class="[letter === ' ' ? 'w-3' : '', className]"
+                class="font-mono inline-block"
+                v-motion
+                :initial="{ opacity: 0, y: -10 }"
+                :enter="{ opacity: 1, y: 0 }"
+                :delay="i * (duration / (text.length * 10))"
+            >
+                {{ letter.toUpperCase() }}
+            </span>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+const props = defineProps({
+    text: {
+        type: String,
+        required: true,
+    },
+    duration: {
+        type: Number,
+        default: 800,
+    },
+    className: {
```

Comment:
`className` is the part of the react. You can make it class and then use $props.class with `cn` utility to apply to the element you want.
Please refer to other component

Reply:
Done.